src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -201,16 +205,19 @@ export abstract class InstructorSessionBasePageComponent {
    * Submits the feedback session as instructor.
    */
   submitSessionAsInstructor(model: SessionsTableRowModel): void {
-    this.router.navigateByUrl('/web/instructor/sessions/submission'
-        + `?courseid=${model.feedbackSession.courseId}&fsname=${model.feedbackSession.feedbackSessionName}`);
+    this.navigationService.navigateByURLWithParamEncoding(
+        this.router,
+        '/web/instructor/sessions/submission',
+        { courseid: model.feedbackSession.courseId, fsname: model.feedbackSession.feedbackSessionName });
   }
 
   /**
    * Views the result of a feedback session.
    */
   viewSessionResult(model: SessionsTableRowModel): void {
     this.router.navigateByUrl('/web/instructor/sessions/result'
-        + `?courseid=${model.feedbackSession.courseId}&fsname=${model.feedbackSession.feedbackSessionName}`);
+        + `?courseid=${model.feedbackSession.courseId}`
+        + `&fsname=${encodeURIComponent(model.feedbackSession.feedbackSessionName)}`);
```

Comment:
Is there a reason why this is done using `encodeURIComponent` instead of `this.navigationService.navigateByURLWithParamEncoding`?

Reply:
Hmm yes forgot to remove this sorry 😨